internal/kafka/avail_status_msg.go

```diff
@@ -44,6 +44,7 @@ func genericMessage(ctx context.Context, m any, key string, topic string) (Gener
 			"x-RH-Identity":               identity.GetIdentityHeader(ctx),
 			"X-RH-Sources-Org-Id":         id.Identity.OrgID,
 			"X-RH-Sources-Account-Number": id.Identity.AccountNumber,
+			"Event_Type":                  "availability_status",
```

Comment:
Can you convert ALL headers to lowercase and add a comment to keep all headers in lowercase, it really looks like lower case is what other teams use.